Src/BootCamp.Chapter/Checks.cs

```diff
@@ -9,31 +10,53 @@ public static class Checks
     {
         public static string FindMostPoorPerson(string[] peopleAndBalances)
         {
-            return "";
+            var people = CreatePeopleObjectFromArrayInput(peopleAndBalances);
+
+            string poorestPerson = people.FindPoorestPerson();
+            return poorestPerson;
         }
 
         public static string FindRichestPerson(string[] peopleAndBalances)
         {
-            return "";
+            var people = CreatePeopleObjectFromArrayInput(peopleAndBalances);
+
+            string richestPerson = people.FindRichestPerson();
+            return richestPerson;
         }
 
         public static string FindPersonWithBiggestLoss(string[] peopleAndBalances)
         {
-            return "";
+            var people = CreatePeopleObjectFromArrayInput(peopleAndBalances);
+
+            string biggestLoss = people.FindPersonWithBiggestLoss();
+            return biggestLoss;
         }
 
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            var people = CreatePeopleObjectFromArrayInput(peopleAndBalances);
+
+            string highestEver = people.FindHighestBalanceEver();
+            return highestEver;
         }
 
         public static string Build(string message, in int padding)
         {
-            return "";
+            var testTextTable = new TextTable(message, padding);
+            return testTextTable.Build();
         }
 
         public static void Clean(string file, string outputFile)
         {
+            var fileWithBalances = new FileWithBalances(file);
+            string cleanedContent = fileWithBalances.GetContent();
+            File.WriteAllText(outputFile, cleanedContent);
+        }
+
+        private static PeopleAndBalances CreatePeopleObjectFromArrayInput(string[] peopleAndBalances)
+        {
+            if (peopleAndBalances == null) peopleAndBalances = new string[0];
```

Comment:
This line does 2 different thing, should be split into another line.
Please use brackets in order to explicitly say what happens if the if condition is true.